### DISCUSSION TITLE
Data construction/process

### DISCUSSION THREAD

**Discussion by kresguerra02** [WFRT/verif#122] [General discussion]
Hi, 

I came across with this verif software because I need to produce a performance diagram plots. Upon inspection of the data is composed of the location (lat and lon), and altitude. I would like to ask if the software can read the format date, obs1, obs2, obs3. Also how can we define the threshold when using this program. Thank you.

![Screenshot (75)](https://github.com/WFRT/verif/assets/119469773/07867bc1-053b-4b08-b817-6b06a45497c7)

**Comment by tnipen** [comment]
Thanks for considering Verif! I can try to help you get your data into a format that can be understood by Verif. You will first need to rearrange the data so that you have one observation on each line. Something like this:

```
location lon lat hour obs
98233 121.76 17.65 6 1.60
98233 121.76 17.65 12 8.20
98233 121.76 17.65 18 4.00
98233 121.76 17.65 0 1.20
98324 119.97 15.33 6 0.00
98324 119.97 15.33 12 0.00
```

You will also need three more columns:
- date: This represents the date in the format YYYYMMDD
- fcst: You need forecasts corresponding to the observations. Otherwise you cannot create a performance diagram.
- leadtime: If you are verifying a forecasting system, you will probably have a leadtime (number of hours in the future that the forecast is for).

More information can be found her: https://github.com/WFRT/verif/wiki/Arranging-my-own-data

After you have organized your data into a text file, you can create a performance diagram like this:
`verif yourfile.txt -m performance -r 5`

Here `-r 5` selects a threshold of 5.
